lib/rest-adapter.js

```diff
@@ -425,6 +433,11 @@ RestAdapter.prototype._registerMethodRouteHandlers = function(router,
     // Express 4.x only supports delete
     verb = 'delete';
   }
+
+  // if (fastRouter.canRegister(route, handler)) {
```

Comment:
> So I just want to be clear and say that you are not suggesting to use the function that exists but instead to make sure that the router can handle the route, otherwise pass it to the express router.

Correct.

> Note: this should be done with an `if else` statement instead of just an `if`.

:+1: